.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 8
+script:
+  - node_modules/bs-platform/bin/bsb -make-world
```

Comment:
Ahh apparently TravisCI does have magic to auto run `npm install` (Yick magic).  :-)